### ISSUE TITLE
Competitor research

### ISSUE THREAD

**Issue by Jack5079** [mollersuite/methodsave#8]
- [Fabric](https://fabric.so)
	- [Has a Linux app!](https://dl.todesktop.com/220930m1ahpjvoh)
		- Obviously Electron
		- Black screen on Wayland
		- On X11 I can't login
	- Has a Chrome extension but no Firefox extension
	- Waitlisted (I am in the beta)
	- Workflow isn't fast enough for me to actually use (Maybe because their integrations DON'T SUPPORT LINUX)
	- They pulled a Shlex: They announced "Sessions" which was gonna be some multiplayer thing [(Etcetera happens to be designing one too)](https://github.com/mollersuite/method/issues/5), then removed it from the landing with zero fanfare
	- Their website is made in Framer (website builder)
- [Lazy](https://lazy.so)
	- [Mac only :/](https://lazy.so/download)
	- [TOS](https://lazy.so/terms-and-conditions) and [Privacy Policy](https://lazy.so/privacy-policy) are lorem ipsum
	- Waitlisted
		- I looked in their build manifest to find those links
	- All videos are faked
	- Workflow looks great, has integrations that remind me of Methodsave for Android
	- Good tagline
	- Etcetera is larger than [the Lazy team](https://twitter.com/lazyappHQ/following)
	- Their website is made in Next.js (React metaframework)
- new.computer
	- No details revealed but based on https://twitter.com/jasonyuandesign/status/1673381928953851905 and https://twitter.com/jasonyuandesign/status/1679975264376492033 it sounds like a Methodsave competitor
	- I thought it was gonna be a mercuryOS implementation
	- Waitlisted
		- Their form asks what company you work at
			- For anyone else curious (Maybe they will prioritize companies with multiple waitlisters), I put `Etcetera Development (https://github.com/mollersuite)`
	- Their website is made in Webflow (website builder)

**Comment by Jack5079** [comment]
![Screenshot_20230826_000434](https://github.com/mollersuite/methodsave/assets/29169102/e52a51b0-6964-4a0e-8d45-c757ed5140c9)

**Comment by Jack5079** [comment]
Lazy is paid LOL

**Comment by nbitzz** [comment]
when will we develop a lazy crack 🗣️ 🗣️

**Comment by nbitzz** [comment]
also isn't lazy electron but still maconly

**Comment by Jack5079** [comment]
> also isn't lazy electron but still maconly

Yes, their website has an unreasonable amount of React dedicated to the "demo" quick capture so I think they just embedded the real thing
